tsconfig.json

```diff
@@ -7,9 +7,9 @@
     "jsxFactory": "React.createElement",
     "moduleResolution": "node",
     "noImplicitAny": true,
```

Comment:
BTW shouldn't we get rid of this `noImplicitAny` line?
AFAIK  `"strict": true` includes this statement:
![image](https://user-images.githubusercontent.com/206567/92580870-9a3e9e80-f297-11ea-8de4-479420b0ca11.png)

Reply:
Yeah, that sounds right. 

It was just a copy/pasted tsconfig that I later added `strict` to.